modules/olm-arch-olm-operator.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-arch.adoc
+
+[id="olm-arch-olm-operator_{context}"]
+= OLM Operator
+
+The OLM Operator is responsible for deploying applications defined by CSV
+resources after the required resources specified in the CSV are present in the
+cluster.
+
+The OLM Operator is not concerned with the creation of the required resources;
+users can choose to manually create these resources using the CLI, or users can
+choose to create these resources using the Catalog Operator. This separation of
+concern allows users incremental buy-in in terms of how much of the OLM
+framework they choose to leverage for their application.
+
+While the OLM Operator is often configured to watch all namespaces, it can also
```

Comment:
I don't think we should suggest this as an option anymore - it will lead to more pain than gain.

Reply:
Paragraph removed.